modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,244 @@
+= APIkit for GraphQL Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configurations
+---
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
+| API definition | String |  |  | *x*
+| Introspection Enabled | Boolean |  Enables schema introspection. |  false | 
+| Max Query Depth Allowed | Number |  The maximum depth a query can have. The value must be greater than 1. |  100 | 
+| Max Query Complexity Allowed | Number | The maximum number of data fields a query can include. The value must be greater than 1. |  100 | 
+| Name | String |  The identifier of this element is used to reference it in other components. |  | *x*
+|===
+
+==== Associated Operations
+
+* <<Route>> 
+* <<Serialize>> 
+* <<SerializeBatch>> 
```

Comment:
Should this bullet be linked to the "=== SerializeBatch" title?
If so, we link to content with an xref: xref:filename.adoc#articletitle[title or phrase to explain title action]

Reply:
OK! Thanks!